src/routes/import-request/add.ts

```diff
@@ -0,0 +1,34 @@
+import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+import { User } from '../../models/user';
+import { FileToServer, schema as fileToServer } from '../../models/file-to-server';
+import {
+  ImportRequest,
+  importRequestSchema,
+} from '../../models/import-request';
+
+type Request = {
+  body: Required<FileToServer>;
+}
+
+type Response = (ImportRequest) & {
+  createdByUsername: User['name'];
+};
```

Comment:
This does not match the Joi schema. In other routes we don't extend models, but use different object properties, see eg. https://github.com/withthegrid/platform-sdk/blob/88852b1e88474d9511f91faa9ab1984f326579a6/src/routes/command/get.ts#L27-L29.

Also occurs at other routes

Reply:
check